lib/features/modeling/ChoreoModeling.js

```diff
@@ -153,15 +151,19 @@ ChoreoModeling.prototype.unlinkCallChoreoParticipants = function(element) {
  * @param resizingShapes
  * @param delta
  * @param direction
+ * @param start
  */
-ChoreoModeling.prototype.createSpace = function(movingShapes, resizingShapes, delta, direction) {
-  movingShapes = movingShapes.filter(s => !is(s, 'bpmn:Participant'));
+ChoreoModeling.prototype.createSpace = function(movingShapes, resizingShapes, delta, direction, start) {
+  if (direction === 'n' || direction === 's') {
+    movingShapes = movingShapes.filter(s => !is(s, 'bpmn:Participant'));
+  }
```

Comment:
Nice! This has been a bug for a while now. It still breaks if you "grab" between messages and their tasks.

![bug](https://user-images.githubusercontent.com/17142708/81398045-c222c780-9128-11ea-87ac-c8279bf8b246.gif)

**We should put this in a new issue and PR, though.**

Reply:
Yes. There are some other things that also still behave strangely with the spacetool. In bpmn-js it is, for example, not possible to resize activities with it. I think we should do the same for chor-js. And allow resizing just for sub-choreos. I will open a new pr on this.